src/components/VErrorImage/meta/VErrorSection.stories.mdx

```diff
@@ -0,0 +1,88 @@
+import {
+  ArgsTable,
+  Canvas,
+  Description,
+  Meta,
+  Story,
+} from '@storybook/addon-docs'
+
+import VErrorSection from '~/components/VErrorImage/VErrorSection.vue'
+import VErrorImage from '~/components/VErrorImage/VErrorImage.vue'
+import VNoResults from '~/components/VErrorImage/VNoResults.vue'
+import VServerTimeout from '~/components/VErrorImage/VServerTimeout.vue'
+
+import { NO_RESULT, SERVER_TIMEOUT } from '~/constants/errors'
+
+<Meta title="Components/VErrorSection" component={VErrorSection} />
+
+# VErrorSection
+
+<Description of={VErrorSection} />
+
+<ArgsTable of={VErrorSection} />
+
+## No results
+
+This section appears when a search returns no hits.
+
+export const NoResultsTemplate = (args, { argTypes }) => ({
+  template: `
+    <VErrorSection>
+      <template #image>
+       <VErrorImage :error-code=errorCode />
```

Comment:
```suggestion
       <VErrorImage :error-code="errorCode" />
```

Reply:
Even if it works, the quotes are not supposed to be omitted so this is a good catch, thanks! Fixed in 9129094.